testing-requirements.txt

```diff
@@ -10,3 +10,4 @@ pytest-cov;          python_version > '2.6'
 pytest-flakes;       python_version > '2.6'
 pytest-pep8;         python_version > '2.6'
 hypothesis>=3.8.0;   python_version > '2.6'
+astroid==1.5.3;      python_version > '2.6'
```

Comment:
What is astroid used for?  I don't see it imported anywhere (although I may have just missed it).